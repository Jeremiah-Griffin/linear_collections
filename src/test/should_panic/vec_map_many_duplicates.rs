pub fn main() {
    linear_collections_macros::vec_map![(0, "v"), (0, "v"), (1, "v"), (1, "v")];
}
