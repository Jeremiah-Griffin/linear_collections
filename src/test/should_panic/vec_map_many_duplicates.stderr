error: proc macro panicked
 --> src/test/should_panic/vec_map_many_duplicates.rs:2:5
  |
2 |     linear_collections_macros::vec_map![(0, "v"), (0, "v"), (1, "v"), (1, "v")];
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Duplicate keys found: ["0", "1"]
